src/OsmMom.sol

```diff
@@ -1,6 +1,8 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
 /// OsmMom -- governance interface for the OSM
 
-// Copyright (C) 2019 Maker Ecosystem Growth Holdings, INC.
+// Copyright (C) 2019-2022 Dai Foundation
```

Comment:
```suggestion
// Copyright (C) 2021-2022 Dai Foundation
```

Reply:
I don't think we need to change the years of the contract. Who was the copyrighter between 2019 and 2021 then?

Reply:
just double checked, you're right (see [copyright guidance](https://github.com/daifoundation/docs/blob/master/COPYRIGHT-GUIDANCE.md))

Reply:
updated previous spdx license suggestions.